graph/src/components/store/traits.rs

```diff
@@ -338,16 +338,15 @@ pub trait WritableStore: ReadStore + DeploymentCursorTracker {
         deterministic_errors: Vec<SubgraphError>,
         offchain_to_remove: Vec<StoredDynamicDataSource>,
         is_non_fatal_errors_active: bool,
+        is_caught_up_with_chain_head: bool,
```

Comment:
Maybe worth commenting what behavior this triggers

Reply:
Documented